packages/slider/stories/slider.stories.ts

```diff
@@ -97,12 +145,234 @@ export const Disabled = (): TemplateResult => {
+                    min="previous"
+                    max="next"
+                ></sp-slider-handle slot="handle">
+                <sp-slider-handle slot="handle"
+                    name="high"
+                    value="250"
+                    min="previous"
+                ></sp-slider-handle slot="handle">
+            </sp-slider>
+        </div>
+    `;
+};
+ThreeHandlesOrdered.args = {
+    tickStep: 10,
+};
+
+// This is a very complex example from an actual application.
+export const ThreeHandlesComplex = (args: StoryArgs): TemplateResult => {
+    const values: { [key: string]: number } = {
+        black: 0,
+        gray: 1.0,
+        white: 255,
+    };
+    const handleEvent = (event: Event): void => {
+        const target = event.target as SliderHandle;
+        if (target.value != null) {
+            if (typeof target.value === 'object') {
+                action(event.type)(target.value);
+            } else {
+                action(event.type)(`${target.name}: ${target.value}`);
+            }
+            values[target.name] = target.value;
+        }
+    };
+    const grayNormalization = {
+        toNormalized(value: number) {
+            const normalizedBlack = values.black / 255;
+            const normalizedWhite = values.white / 255;
+            const clamped = Math.max(Math.min(value, 1), 0);
+            return (
+                clamped * (normalizedWhite - normalizedBlack) + normalizedBlack
+            );
+        },
+        fromNormalized(value: number) {
+            const normalizedBlack = values.black / 255;
+            const normalizedWhite = values.white / 255;
+            const clamped = Math.max(
+                Math.min(value, normalizedWhite),
+                normalizedBlack
+            );
+
+            return (
+                (clamped - normalizedBlack) /
+                (normalizedWhite - normalizedBlack)
+            );
+        },
+    };
+    const blackNormalization = {
+        toNormalized(value: number) {
+            const clamped = Math.min(value, values.white);
+            return clamped / 255;
+        },
+        fromNormalized(value: number) {
+            const denormalized = value * 255;
+            return Math.min(denormalized, values.white);
+        },
+    };
+    const whiteNormalization = {
+        toNormalized(value: number) {
+            const clamped = Math.max(value, values.black);
+            return clamped / 255;
+        },
+        fromNormalized(value: number) {
+            const denormalized = value * 255;
+            return Math.max(denormalized, values.black);
+        },
+    };
+    const computeGray = (value: number): string => {
+        let result = 1.0;
+        if (value > 0.5) {
+            result = Math.max(2 * (1 - value), 0.01);
+        } else if (value < 0.5) {
+            result = ((1 - 2 * value) * (Math.sqrt(9.99) - 1) + 1) ** 2;
+        }
+        const formatOptions = {
+            maximumFractionDigits: 2,
```

Comment:
You might want `minimumFractionDigits: 2` here as well so that the text doesn't jump around so much. Though I guess that makes it display `1.00` vs `1` which might also not be appropriate.

Reply:
Yep, that looks better...